packages/javalib.2.3.1/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.1.1"
+name: "javalib"
+maintainer: "sawja@inria.fr"
+version: "2.3.1"
```

Comment:
Could you please remove `name` and `version` when publishing? It helps keep the repository file reusable for variants.